src/pages/home/report/PureReportActionItem.tsx

```diff
@@ -834,7 +835,9 @@ function PureReportActionItem({
 
             // Table Report View does not display these components as separate messages, except for self-DM
             if (canUseTableReportView && report?.type === CONST.REPORT.TYPE.CHAT) {
-                if ((report.chatType === CONST.REPORT.CHAT_TYPE.SELF_DM && !isDeletedAction(action)) || isSplitInGroupChat) {
+                const isSplitBill = moneyRequestActionType === CONST.IOU.REPORT_ACTION_TYPE.SPLIT;
+                const shouldShowSplitBill = !(isDM(report) || isPolicyExpenseChat(report));
```

Comment:
Came here from some other issue, but we want to keep showing the split preview as a separate action in dms and expense chats cc @trjExpensify

Reply:
Agree.

Reply:
Yeah I mean to some extent they should not be added to the chats at all, but we can also make sure not to show them in such reports in FE